lib/dialogs/logout-confirmation/index.tsx

```diff
@@ -32,56 +34,45 @@ export class LogoutConfirmation extends Component<Props> {
 
     return (
       <div className="logoutConfirmation">
-        <Dialog onDone={closeDialog} title="Unsynchronized Notes Found">
-          <section>
-            <h1>Are you sure you want to logout?</h1>
-          </section>
-
+        <Dialog onDone={closeDialog} title="Unsynced Notes">
+          <p className="explanation">
+            {notes.size > 0
+              ? 'Logging out will delete any unsynced notes.'
+              : 'All notes have syncronized!'}
```

Comment:
typo: "synchronized" (also one of the reasons I don't like `unsynced` and `synced` is because of the missing `h` - this is why I prefer the other short-hand version `sync'd` which opaquely shows we're removing letters vs. `synced` which makes it look like we aren't when in fact we are)

Reply:
done